hpy/devel/include/common/hpytype.h

```diff
@@ -33,7 +33,7 @@ typedef struct {
 
 typedef enum {
     HPyType_SpecParam_Base = 1,
-    HPyType_SpecParam_Bases = 2,
+    HPyType_SpecParam_BasesTuple = 2,
```

Comment:
do we really need/want `BasesTuple`? Can't we just use `Base`?

Reply:
I don't see a problem to support ``BasesTuple`` and it's closer to what is done in the CPython C API.  It avoids a potentially messy deconstruction of the tuple of bases in user code if he starts with that.